cmd/removeEnumerator.go

```diff
@@ -109,3 +117,89 @@ func (s *directoryStack) Pop() (*azfile.DirectoryURL, bool) {
 		return &e, true
 	}
 }
+
+// TODO move after ADLS/Blob interop goes public
+// TODO this simple remove command is only here to support the scenario temporarily
+// Ultimately, this code can be merged into the newRemoveBlobEnumerator
+func removeBfsResource(cca *cookedCopyCmdArgs) (successMessage string, err error) {
+	ctx := context.Background()
+
+	// return an error if the unsupported options are passed in
+	if len(cca.includePatterns)+len(cca.excludePatterns) > 0 {
```

Comment:
...Shouldn't this be in cca.cook()?

Though, then again, this code is meant to be removed. May as well just leave this here so it's easier to get rid of when the time comes.

Reply:
that's what I thought too, it'd be good to put these things all together

Reply:
the things to be removed later